config/mt793x/args.gni

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2022 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Options from standalone-chip.mk that differ from configure defaults. These
+# options are used from examples/.
+
+#chip_device_platform = "mt793x"
+
+chip_project_config_include = ""
+chip_system_project_config_include = ""
+chip_ble_project_config_include = ""
+
+mbedtls_target = "//mbedtls:mbedtls"
+lwip_platform = "external"
+
+chip_build_tests = true
+
+#Enabling this causes some error
```

Comment:
Maybe add error text in here for reference? 'some error' is very vague.

Reply:
As I know, this comment is due to the problems encountered at early development stage, it should have been solved!
These code are removed in https://github.com/project-chip/connectedhomeip/pull/22404/commits/ea7b5c691632505ef116a0c03acb1a26d921abac